### PR TITLE
Gaffer docker container for Gaffer version 0.3.9.  

### PR DESCRIPTION
**Pull request by cybermaggedon**
The container builds on my Accumulo/Zookeeper/Hadoop containers, so that Gaffer launches from a
single container.  This container also includes Wildfly, you should download wildfly-10.1.0.CR1.zip to the local directory before building the container.

This includes two PATCH files to allow the Gaffer build to work with
OpenJDK, and also includes a basic default schema.

### CODE REVIEW

**Comment by cybermaggedon** [comment]
Updated with repository push (make push).  Tested and works.  This pushes to my repo, suggest you change the repository name with your repo in the Makefile.

File: Dockerfile.deploy [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+WORKDIR /usr/local/wildfly
+
+COPY product/example-rest-*.war standalone/deployments/example-rest.war
+COPY product/*.jar /usr/local/accumulo/lib/ext/
+COPY standalone.xml standalone/configuration/
+RUN mkdir schema
+COPY product/*.json schema/
+COPY product/mockaccumulostore.properties store.properties
+
+CMD cd /; /start-accumulo; cd /usr/local/wildfly; bin/standalone.sh -Dgaffer.rest-api.basePath=/example-rest/v1 -Dgaffer.schemas=/usr/local/wildfly/schemas -Dgaffer.storeProperties=/usr/local/wildfly/store.properties
```
Comment (p013570) [inline Dockerfile.deploy]: There is a typo in the schema directory property. It should be 'schema' not 'schemas':
-Dgaffer.schemas=/usr/local/wildfly/schema

**Comment by cybermaggedon** [comment]
Fixed the typo, and added a README. Any good?

**Comment by p013570** [comment]
Fantastic - thanks!

**Pull request merged**